codec/src/lib.rs

```diff
@@ -822,6 +788,7 @@ pub struct WindowTitleChanged {
 #[derive(Deserialize, Serialize, PartialEq, Debug)]
 pub struct PaneFocused {
     pub pane_id: PaneId,
+    pub pane_focus_serial: Option<InputSerial>,
```

Comment:
the various changes in this file require that we bump the codec version:

https://github.com/wezterm/wezterm/blob/364a158b6c615ee0d5cdfcd0ffa83fe73714ecdf/codec/src/lib.rs#L441-L444